index.cjs

```diff
@@ -24,6 +23,13 @@ const shelljs = require('shelljs'),
 	chalk = require('chalk'),
 	matter = require('gray-matter'),
 	parseArgs = require('minimist');
+const documentation = import('documentation');
+
+let documentationResponse;
+const generateDocumentationResponse = async () => {
+	documentationResponse = await documentation.then(result => result);
+	return documentationResponse.build;
+};
```

Comment:
Dynamic import works fine but it's a workaround way.
I think we need to consider `ESM` refactoring.

Could you check this project to change the `ESM` module?
Please check the scale of the work and check whether it can be modified.

Reply:
I get some errors on `npm run parse` when I load some modules with `import`.
I think it would be best to do ESM refactoring in a separate ticket

Reply:
OK, Could you please create a new ticket? :)